### PR TITLE
Dv Team Blog Post

### PR DESCRIPTION
**Pull request by violetadev**
dv team blogpost

### CODE REVIEW

File: blog/_posts/2018-07-09-dvteam-activated.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: DV Team Activated!
+layout: post
+created_at: Mon Jul 09 2018
+permalink: blog/dvteam_activated
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-dvteam-activated.md]: Hi @artnerdnet ! Thanks for the PR. Please check the comments below.

Permalink should have same file name as your markdown file [along with date]. You can check the other posts for example.

File: blog/_posts/2018-07-09-dvteam-activated.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+title: DV Team Activated!
+layout: post
+created_at: Mon Jul 09 2018
+permalink: blog/dvteam_activated
+current: blog
+author: Violeta
+twitter: artnerdnet
+categories:
+- news
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-dvteam-activated.md]: Please change the categories to 'student-posts'

File: blog/_posts/2018-07-09-dvteam-activated.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+
+We started with a selected group of 20.000 out of 200.000 applications in Europe, Africa and Middle East region. Earning a full Google Scholarship for Front End Web Developer at Udacity was the fruit of this three months of dedication, hard work, discipline, cooperation with peers coders and taking actively part in a huge social community, in order to stand out and win our spot in the top 2.000 students (out of 20.000). We are now about to finish the last part: React.
+
+### RgSoC Coaches & Supervisor
+During the Google Scholarship, we met several times in real life and decided to apply for RgSoc to be able to keep practicing our coding skills in summer. As we were new in the dev community, we reached out to our Udacity community managers who introduced us to Amanda Cavallaro. She helped us find our first two current RgSoc-coaches: 
+
+*Coach 1: Tobias (React developer)*
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-dvteam-activated.md:24]: All coaches have some intro sort of text except this one. May be for the consistency purpose, it makes sense to add some lines for this coach too. :)

File: blog/_posts/2018-07-09-dvteam-activated.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+The last RgSoc-coach to join, we met during a recently peer lab, that Violeta organized for Woman Who Code. We really liked her spirits and clarity so just a week before starting the SoC we invited her to be part of it. 
+
+*Coach 4: Cristina (full stack developer)*
+
+We consider ourselves very lucky! As not only do we have gathered a great team of mentors, but also an interesting supervisor Lucas Pinto who has even organized Ruby conferences! 
+
+### Living Style Guide
+The Open Source project we have been selected for is Living Style Guide, which helps developers to create front-end style guides easily with SaSS and Markdown. The project is written by Nico Hagenburger in Ruby. It needs to be rewritten to and tested for a Vanilla Javascript version, which will be our main tasks, beside improving the documentation. 
+
+Summer and future Goals
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-dvteam-activated.md]: This looks like some title, so may be it makes sense to add some style or colon here. Add one blank line too. At the moments it is just the same sentence which looks weird.

File: blog/_posts/2018-07-09-dvteam-activated.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+##### Twitter
+[Violeta](http://www.twitter.com/artnerdnet)
+
+##### Github
+[Living Style Guide](https://github.com/livingstyleguide/livingstyleguide)
+[Violeta](http://www.github.com/artnerdnet)
+[Diana](http://www.github.com/dianavile)
+
+# For a great beginning!
+![image](https://media.giphy.com/media/WeJeWpUMxpQmk/giphy.gif)
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-dvteam-activated.md]: Please add credits using image-credits class.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-07-09-dvteam-activated.md (hunk 5 above)
Comment (violetadev) [reply blog/_posts/2018-07-09-dvteam-activated.md]: not sure if I did it correctly

**Review by violetadev — COMMENTED** [review]
Please check the image credit

**Pull request closed without merging**